src/ctia/http/routes/common.clj

```diff
@@ -134,3 +134,47 @@
     true {:refresh "wait_for"}
     false {:refresh "false"}
     {}))
+
+(s/defschema Capability
+  (s/conditional
+    keyword? (s/pred simple-keyword?)
+    nil? (s/pred nil?)
+    set? #{(s/pred simple-keyword?)}))
+
+(s/defn capabilities->string :- s/Str
+  "Does not add leading or trailing new lines."
+  [capabilities :- Capability]
+  (cond
+    (keyword? capabilities) (name capabilities)
+    ((every-pred set? seq) capabilities) (->> capabilities
+                                              sort
+                                              (map name)
+                                              (str/join ", "))
+    :else (throw (ex-info "Missing capabilities!" {}))))
+
+(s/defn capabilities->description :- s/Str
+  "Does not add leading or trailing new lines."
+  [capabilities :- Capability]
+  (cond
+    (keyword? capabilities) (str "Requires capability " (capabilities->string capabilities) ".")
+    ((every-pred set? seq) capabilities) (str "Requires capabilities " (capabilities->string capabilities) ".")
+    :else (throw (ex-info "Missing capabilities!" {}))))
+
+(defmacro reloadable-function
+  "Transforms v to (var v).
+
+  This can help REPL development in several situations.
+
+  When entities are top-level maps, :services->routes
+  is fixed when the entity is evaluated.
+  If passing :services->routes using a var deref,
+  this means updates to the route function will
+  not be observed until *both* the entity namespace and
+  the server's routes have been reloaded.
+
+  Using (var v) instead of v in top-level
+  maps enables routes to dynamically
+  update."
+  [v]
+  {:pre [(symbol? v)]}
+  `(var ~v))
```

Comment:
I don't get how writing `(reloadable-function f)` is different from writing `#'f`?

Reply:
It prevents having to respond to future questions about why var is used here.

Reply:
👍it's not perfect (since one normally doesn't alias clojure.core fns to denote intent) but it works.

Maybe the `reloadable-function` docstring could be clearer in reflecting that it doesn't do something special.

Or as an alternative:

`^:do-not-unwrap-var #'foo, ^:do-not-unwrap-var #'bar` etc. git log/blame would do the rest

Reply:
I concluded that since 3 team members asked me about it, that using var here is "special enough" to warrant this. Obviously I didn't start this PR with `reloadable-function`.

You can add meta to vars with that syntax? I thought that only worked with IObjs.